cmd/podman/common/specgen.go

```diff
@@ -655,26 +655,53 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *ContainerCLIOpts, args []string
 	return nil
 }
 
-func makeHealthCheckFromCli(inCmd, interval string, retries uint, timeout, startPeriod string) (*manifest.Schema2HealthConfig, error) {
+func makeHealthCheckFromCli(inCmd string, interval string, retries uint, timeout, startPeriod string) (*manifest.Schema2HealthConfig, error) {
```

Comment:
Why was the signature changed? Looks like addition of duplicate `string` specifiers, nothing functional

Reply:
Typo sorry. I was originally doing `[]string` on the cmd level but changed it back to `string`